site/docs/src/json/scim/scim-schemas-response.json

```diff
@@ -0,0 +1,208 @@
+{
+  "itemsPerPage": 4,
```

Comment:
Should be alphabetized. jq does it well.

Reply:
done